examples/compiled/area_density_stacked_fold.vg.json

```diff
@@ -24,7 +24,8 @@
           "extent": [10, 500],
           "counts": true,
           "steps": 200,
-          "as": ["value", "density"]
+          "as": ["value", "density"],
+          "resolve": "shared"
```

Comment:
Should this one change?

Reply:
Yeah, the input Vega-Lite spec has:

```json
    {
      "density": "value",
      "bandwidth": 10,
      "groupby": ["Measurement"],
      "extent": [10, 500],
      "counts": true,
      "steps": 200
    }
```
Since `groupby` is defined we get `"resolve": "shared"`. `"steps": 200` is still there because it's explicitly specified in the Vega-Lite transform.